README.md

```diff
@@ -82,7 +82,7 @@ If you are encountering problems, the following tips may help in troubleshooting
 
 - If content is missing from your transformed article, more likely than not there isn't a **Transformer Rule** matching an element in your source markup. See how to configure appropriate rules for your content in the [Transformer Rules documentation](https://developers.facebook.com/docs/instant-articles/sdk/transformer-rules).
 
-- Set the `threshold` in the [configuration of the Logger](https://logging.apache.org/log4php/docs/configuration.html#PHP) to `DEBUG` to expose more details about the items processed by the Transformer.
+- Set the `threshold` in the transformer Logger level to `DEBUG` to expose more details about the items processed by the Transformer. `$transformer->setLogLevel(Transformer::LOG_DEBUG);`
```

Comment:
Just Curious, What will be the advantage of using Logger implemented within the same code, instead of using the log4php library which already has lot of functionality.